bitnami/keycloak/values.yaml

```diff
@@ -266,6 +266,9 @@ extraEnvVarsSecret: ""
 ## @param replicaCount Number of Keycloak replicas to deploy
 ##
 replicaCount: 1
+## @param revisionHistoryLimitCount Number of controller revisions to keep
+##
+revisionHistoryLimitCount: 3
```

Comment:
We should leave this value at `10` given it's the default one and changing this here would modify the current behaviour.